docs/graphql/manual/how-it-works/index.rst

```diff
@@ -0,0 +1,70 @@
+How Hasura GraphQL engine works
+===============================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Given your Postgres database, Hasura GraphQL engine can automatically generate a GraphQL schema and process GraphQL
+queries, subscriptions and mutations. Here’s what Hasura GraphQL engine does under the hood.
+
+Schema generation
+-----------------
+
+Hasura GraphQL engine automatically generates GraphQL schema components when you create/track a
```

Comment:
this tracking business is not PG related no? It may be confusing. Either say "create in PG and track in Hasura" or just say "create" throughout this section and add a note afterwards that these tables need to be created. Same goes for the word "relationships" - not a PG thingy.